package.json

```diff
@@ -26,7 +26,8 @@
   },
   "devDependencies": {
     "jshint": "*",
-    "mocha": "*"
+    "mocha": "*",
+    "fs-extra": "*"
```

Comment:
I only use fs-extra because I thought `outputFileSync` was cleaner then having to create the directories. 

I can change it, however, I think rob brought up a great point. I'd prefer to go his route which would allow me to use the actual index.html file.